.github/dependabot.yml

```diff
@@ -97,6 +97,17 @@ updates:
     commit-message:
       prefix: "deps"
       include: "scope"
+  - package-ecosystem: "gomod"
```

Comment:
I think we need an entry for the `processor/metricextractprocessor` module also.